Split ant cmdline arguments to avoid "Unknown argument: -lib /path"

cryptogopher opened the pull request:
errors.

This is necessary when providing -lib paths through config.build ANT_OPTIONS

cryptogopher committed:
Split ant cmdline arguments to avoid "Unknown argument: -lib /path"
errors.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Zimbra/zm-build?pullRequest=134) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/Zimbra/zm-build?pullRequest=134) before we can accept your contribution.<br/>**11** out of **12** committers have signed the CLA.<br/><br/>:white_check_mark: umagmrit<br/>:white_check_mark: sneha-patil-synacor<br/>:white_check_mark: rupalid<br/>:white_check_mark: cryptogopher<br/>:white_check_mark: k-kato<br/>:white_check_mark: nileshpawar1<br/>:white_check_mark: sitaram-mulik-synacor<br/>:white_check_mark: silentsakky<br/>:white_check_mark: robert-scheck<br/>:white_check_mark: dasiyogesh<br/>:white_check_mark: GaneshAnarse<br/>:x: gautamdg<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Zimbra/zm-build?pullRequest=134) it.</sub>

cryptogopher committed:
Merge tag '8.8.10' into develop

release 8.8.10 8.8.10

cryptogopher committed:
Added GENTOO.

cryptogopher committed:
Added config to git.

cryptogopher committed:
Working version.

sneha-patil-synacor committed:
ZBUG-1148:Loosing table format with owasp sanitizer

sneha-patil-synacor committed:
adding remote for owasp repo entry

nileshpawar1 committed:
ZBUG-1174: Web client does not show UTF-8 "Quoted-Printable" (#157)

* ZBUG-1174: Web client does not show UTF-8 "Quoted-Printable"

* ZBUG-1174: used existing remote

umagmrit committed:
Fix for ZCS-7871

Commit:
Use HTTPS instead of git protocol for clone
- Revert partial changes from https://github.com/Zimbra/zm-build/commit/4b683fd0d55e403bc172a8e2e0f04e315b7c0ca6
and https://github.com/Zimbra/zm-build/commit/de78c9053ff4cfbd8094f704634798f6abd26140
- Change makes sure external contributors can create FOSS build without configuring ssh authentication for github

umagmrit committed:
ZCS-9017: Changes for RHEL8

rupalid committed:
ZBUG-148 Fixing Tika extractor (#172)

(cherry picked from commit 0c71672011b34651835754d2763bffa63f275dc8)
(cherry picked from commit 6547156ed3e5c082b28bf564b6032d3f02862559)

umagmrit committed:
syntax fix for ZBUG-1479

umagmrit committed:
Fix for ZCS-9406

umagmrit committed:
ZCS-11781:exclude backup directory from zco builds

umagmrit committed:
ZCS-10795: Changes for UBUNTU20

umagmrit committed:
ZCS-10795: Changes for UBUNTU20

umagmrit committed:
ZCS-10795: Changes for UBUNTU20

robert-scheck committed:
Add Rocky Linux 8

CentOS Linux 8 reaches EOL on 2021-12-31 and is superseeded by Rocky
Linux 8 as alternative bug-for-bug compatible RHEL 8 rebuild.

k-kato committed:
Fix for ZCS-11090

dasiyogesh committed:
moving tika library copy jar for convertd ext to network

umagmrit committed:
ZCS-11305:option to choose installation of patch packages in lab machines

umagmrit committed:
ZBUG-2633:Upgraded jetty to 9.4.46.v20220331

umagmrit committed:
ZCS-11813:Add Java options in zmsetup.pl for upgrade

silentsakky committed:
ZBUG-2867 Run nginx and ldap with lesser privileges
- Remove sudo access and change binary permissions to root:zimbra
- Use setcap for ldap and proxy to start services on privileged ports

silentsakky committed:
ZCS-12171 Remove nginx resolvers.conf

silentsakky committed:
ZBUG-2974 Exclude .git directories from build
- zm-help and zm-downloads repos doesn't have any ant target for creating artifacts
- and zm-build repo just copies all files from above repos to include it as part of final build
- but in this inclusion zm-build is using cp command which copies .git directories as well
- so we are excluding .git directories when copying data and creating build

silentsakky committed:
Revert changes of ZBUG-2867

umagmrit committed:
Revert changes of ZBUG-2867:Corrected 02_zimbra-proxy file format

gautamdg committed:
ZBUG-3378: Removed clientuploader zimlet and extension

sitaram-mulik-synacor committed:
ZCS-2733 Removed product help pages from the build

sitaram-mulik-synacor committed:
ZCS-2733 removed other references of zm-help repo which does additional preprocessing

GaneshAnarse committed:
ZCS-13806: Added rclone command to download network build dependencies. (#257)

cryptogopher committed:
Merge tag '8.8.15.p44' into develop and update

cryptogopher committed:
Fix version number

cryptogopher committed:
Update PKG_OS_TAG

silentsakky commented:
Looks like some issue with PR, it's showing unnecessary changes

cryptogopher committed:
Specify git default tag for 8.8.15.p44 patchlevel

cryptogopher committed:
Override git settings for zm-pkg-tool repository

cryptogopher committed:
Fix git remote override

cryptogopher committed:
Fix git remote override

cryptogopher committed:
Change git remote override from ssh to https

cryptogopher committed:
Add modified repository update instructions

cryptogopher committed:
When resuming, continue searching through git tags

cryptogopher committed:
Skip Ant tests

cryptogopher committed:
Move instructions to Docker entrypoint for visibility

cryptogopher commented:
PR created from improper branch
